interface/src/app/app.spec.js

```diff
@@ -1,17 +1,17 @@
-describe( 'AppCtrl', function() {
-  describe( 'isCurrentUrl', function() {
+describe('AppCtrl', function() {
+  describe('isCurrentUrl', function() {
     var AppCtrl, $location, $scope;
 
-    beforeEach( module( 'adage' ) );
+    beforeEach(module('adage'));
 
-    beforeEach( inject( function( $controller, _$location_, $rootScope ) {
+    beforeEach(inject(function($controller, _$location_, $rootScope) {
```

Comment:
Interesting - I had never noticed that the _$location_ in the parameters of this function in this test file had those underscores. I wonder why that is special about the $location service that makes it require that?